lib/tilt/pipeline.rb

```diff
@@ -0,0 +1,60 @@
+require 'tilt'
+
+module Tilt
+  # @see Tilt::Mapping#pipeline
+  def self.pipeline(ext, options={})
+    default_mapping.pipeline(ext, options)
+  end
+
+  class Mapping
```

Comment:
These could go in their respective files, no?

Reply:
Yep, moved them to the appropriate files.